examples/spark2/src/main/scala/org/apache/carbondata/examples/S3Example.scala

```diff
@@ -157,8 +157,8 @@ object S3Example {
   }
 
   def getSparkMaster(args: Array[String]): String = {
-      if (args.length == 5) args(4)
-      else if (args(3).contains("spark:") || args(3).contains("mesos:")) args(3)
-      else "local"
-    }
```

Comment:
revert the unnecessary changes

Reply:
the format is incorrect. we should optimize it.
```def getS3EndPoint(args: Array[String]): String = {		   def getS3EndPoint(args: Array[String]): String = {
     if (args.length >= 4 && args(3).contains(".com")) args(3)		     if (args.length >= 4 && args(3).contains(".com")) args(3)
     else ""		     else ""
   }```